### PR TITLE
#12/home page impact section

### PR DESCRIPTION
**Pull request by jeffoty**
# Description

I worked on this issue #12 with @Allan-Ngugi to create Home page impact section feature and its responsiveness.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-02-17 11-25-16

Screenshot from 2023-02-17 22-57-17

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 1]
```diff
@@ -16,24 +16,28 @@ import { NextDirective } from './directives/next.directive';
...
+    ButtonsModule
   ],
   declarations: [
     HomePageComponent,
     HomeHeroSectionComponent,
     HomeJobsSectionComponent,
-    TeamAndPartnersComponent,
     ActivitySectionComponent,
-    NextDirective,
-    PrevDirective
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:35]: This directives are being used by the values carousel on mobile, should not be removed.

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 2]
```diff
@@ -16,24 +16,28 @@ import { NextDirective } from './directives/next.directive';
...
+
 
 @NgModule({
   imports: [
     CommonModule,
     LayoutModule,
     NgOptimizedImage,
     CardsModule,
-    HomeRoutingModule
+    HomeRoutingModule,
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:30]: Router should be last on imports

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts (hunk 2 above)
Comment (jeffoty) [reply libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:26]: resolved

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts (hunk 1 above)
Comment (jeffoty) [reply libs/pages/elewa/home/src/lib/features-elewa-home.module.ts]: resolved

**Pull request merged**